### ISSUE TITLE
Design the user interface of the system

### ISSUE THREAD

**Issue by ajaynair** [ajaynair/karam#20]
Initial MockUp: https://docs.google.com/presentation/d/1oI2-cqtVGvDCyyQhp7ustE7U4FWTkWYEDvSsy-Ri0ag/edit?usp=sharing
Comment if anyone wishes to work on this!

**Comment by ramansrivastava** [comment]
hey i am a beginner and want to help can i get assigned?

**Comment by ajaynair** [comment]
Hello @ramansrivastava sure. Would you like to join the slack channel?

**Comment by ramansrivastava** [comment]
yes sure ,but i am a beginner in python(and programming) so you have to tell me what exactly to do.

**Comment by ajaynair** [comment]
That is fine. You can find the link to slack on the readme or just click here: https://join.slack.com/t/karam2/shared_invite/zt-ffltc2x1-_Yn1bEgEQ7wrfVOajxLFgw

**Comment by 28avinash** [comment]
hey ,can I get access to help you in creating UI for your Application?

**Comment by 28avinash** [comment]
I have gone through your UI design and I am sure that I can help you.
